frontend/src/pages/NewRunV2.tsx

```diff
@@ -131,30 +167,38 @@ function NewRunV2(props: NewRunV2Props) {
   const [isStartingNewRun, setIsStartingNewRun] = useState(false);
   const [errorMessage, setErrorMessage] = useState('');
   const [isParameterValid, setIsParameterValid] = useState(false);
-  const [isRecurringRun, setIsRecurringRun] = useState(false);
-  const [trigger, setTrigger] = useState<ApiTrigger>();
-  const [maxConcurrentRuns, setMaxConcurrentRuns] = useState('10');
+  const [isClone, setIsClone] = useState(false);
```

Comment:
`isClone` should not be a state from my opinion, it is because user will enter from different entrypoint if it is a clone, and they cannot select pipeline and pipeline version if it is clone. So `isClone` should be determined by the time of first rendering.